dspy/clients/finetune.py

```diff
@@ -51,28 +51,33 @@ class TrainingStatus(str, Enum):
     TrainingMethod.Preference: ["prompt", "chosen", "rejected"],
 }
 
-
 class FinetuneJob(Future):
 
     def __init__(self,
         model: str,
         train_data: List[Dict[str, Any]],
         train_kwargs: Optional[Dict[str, Any]]=None,
+        provider: str = "openai",
+        method: TrainingMethod = TrainingMethod.SFT,
     ):
         self.model = model
         self.train_data = train_data
         self.train_kwargs: Dict[str, Any] = train_kwargs or {}
+        self.provider = provider
+        self.method = method
         super().__init__()
     
     def get_kwargs(self):
         return dict(
             model=self.model,
             train_data=self.train_data,
             train_kwargs=self.train_kwargs,
+            provider=self.provider,
+            method=self.method,
         )
 
     def __str__(self):
-        return f"FinetuningJob({self.to_dict()})"
+        return f"FinetuningJob({self.model})"
```

Comment:
The older version was useful to get more information about the job, but either works

Reply:
I was getting some issues with it, but I think they were red herrings - reverting